Makefile

```diff
@@ -0,0 +1,64 @@
+
+# Image URL to use all building/pushing image targets
+IMG ?= controller:latest
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+CRD_OPTIONS ?= "crd:trivialVersions=true"
```

Comment:
It might be good to go ahead and not use trivial versions to allow easier upgrades later.

Reply:
i don't even know what that is, i still have to read through this stuff 😆 (hence the hold)

Reply:
Unless you have multiple versions, that flag has to be used for making it work with controller-tools v2

Reply:
https://book.kubebuilder.io/print.html#multiple-versions